### PR TITLE
Create generalized-plasma.md

### PR DESCRIPTION
**Pull request by Ro5s**
Add updated article on generalized Plasma, including reference to Plasma Leap

### CODE REVIEW

**Comment by Ro5s** [comment]
@barrasso Opened new PR for 'generalized Plasma' page (including references to Plasma Leap design). Let me know if this works! 👌

File: source/en/learn/generalized-plasma.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    Plasma MVP: /en/learn/mvp.html
+    Plasma Cash: /en/learn/cash.html
+    Plasma Debit: /en/learn/debit.html
+  after:
+    Comparison: /en/learn/compare.html
+---
+
+## Generalized Plasma
+
+It is difficult to create a Plasma chain that operates a lot like Ethereum does today and is capable of running more general smart contracts. This page seeks to address the question, ["Why is EVM-on-Plasma hard?,"](https://medium.com/@kelvinfichter/why-is-evm-on-plasma-hard-bf2d99c48df7) with an introduction to possible solutions, such as the “Plasma VM” design introduced by researcher Kelvin Fichter and the "Plasma Leap" design being developed by LeapDAO.
```
Comment (kfichter) [inline source/en/learn/generalized-plasma.md]: Would recommend that every sentence be separated into an individual line to make diffing easier when we modify content.

Comment (Ro5s) [reply source/en/learn/generalized-plasma.md]: Sure thing~definitely want ease-of-edits. Will make that tweak now 👍

**Comment by Ro5s** [comment]
thanks for reviewing @kfichter hopefully my tweaks on spacing worked 🔧

**Comment by barrasso** [comment]
Merging. Thanks for the good work @Ro5s

**Pull request merged**